### PR TITLE
[hi] Localize content/hi/docs/reference/glossary/deployment.md

### PR DESCRIPTION
**Pull request by hamees-sayed**
Fixes #35600 Localize content/en/docs/reference/glossary/deployment.md into content/hi/docs/reference/glossary/deployment.md

### CODE REVIEW

**Comment by Kartik494** [comment]
Hi @hamees-sayed could you please update PR description from `Fix Issue #Issuenumber` to `fixes #Issuenumber` so that it will automatically closes the issue  once this PR got merged.
Thanks!

**Comment by amolmote** [comment]
/retitle [hi] Localize content/hi/docs/reference/glossary/deployment.md

**Comment by amolmote** [comment]
> Hi @hamees-sayed could you please update PR description from `Fix Issue #Issuenumber` to `fixes #Issuenumber` so that it will automatically closes the issue once this PR got merged. Thanks!

hi @Kartik494. this is the command issue for whole glossary entries. @hamees-sayed  you can do like this `Fixes#Issue_number` `Localize content/en/docs/reference/glossary/deployment.md into content/hi/docs/reference/glossary/deployment.md`

File: content/hi/docs/reference/glossary/deployment.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+tags:
+- fundamental
+- core-object
+- workload
+---
+ एक API ऑब्जेक्ट जो एक प्रतिकृति एप्लिकेशन का प्रबंधन करता है, आमतौर पर पॉड्स को बिना किसी स्थानीय स्थिति के चलाकर।
+
+<!--more--> 
+
+प्रत्येक प्रतिकृति को एक {{<glossary_tooltip term_id="pod" >}} द्वारा दर्शाया जाता है, और पॉड्स को बीच में वितरित किया जाता है {{< glossary_tooltip text="nodes" term_id="node" >}} एक क्लस्टर का।
```
Comment (amolmote) [inline content/hi/docs/reference/glossary/deployment.md:19]:
```suggestion
प्रत्येक प्रतिकृति को एक {{<glossary_tooltip term_id="pod" >}} द्वारा दर्शाया जाता है, और पॉड्स को एक क्लस्टर के  {{< glossary_tooltip text="नोड्स" term_id="node" >}} के बीच वितरित किया जाता है।
```

File: content/hi/docs/reference/glossary/deployment.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+ऐसे कार्यभार के लिए जिन्हें स्थानीय स्थिति की आवश्यकता होती है, {{< glossary_tooltip term_id="StatefulSet" >}} का उपयोग करने पर विचार करें।
```
Comment (amolmote) [inline content/hi/docs/reference/glossary/deployment.md:20]:
```suggestion
ऐसे वर्कलोड के लिए जिन्हें स्थानीय स्थिति की आवश्यकता होती है, {{< glossary_tooltip term_id="StatefulSet" >}} का उपयोग करने पर विचार करें।
```

**Review by amolmote — CHANGES_REQUESTED** [review]

File: content/hi/docs/reference/glossary/deployment.md (hunk 2 above)
Comment (hamees-sayed) [reply content/hi/docs/reference/glossary/deployment.md:20]: understood!

File: content/hi/docs/reference/glossary/deployment.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+- fundamental
+- core-object
+- workload
+---
+ एक API ऑब्जेक्ट जो एक प्रतिकृति एप्लिकेशन का प्रबंधन करता है, आमतौर पर पॉड्स को बिना किसी स्थानीय स्थिति के चलाकर।
+
+<!--more--> 
+
+प्रत्येक प्रतिकृति को एक {{<glossary_tooltip term_id="pod" >}} द्वारा दर्शाया जाता है, और पॉड्स को एक क्लस्टर के {{< glossary_tooltip text="नोड्स" term_id="node" >}} के बीच वितरित किया जाता है।
+ऐसे वर्कलोड के लिए जिन्हें स्थानीय स्थिति की आवश्यकता होती है, {{< glossary_tooltip term_id="StatefulSet" >}} का उपयोग करने पर विचार करें।
```
Comment (Babapool) [inline content/hi/docs/reference/glossary/deployment.md:20]: I think it is better to transliterate `local state` as readers might not be able to associate with `स्थानीय स्थिति` easily. Thoughts, WDYT @hamees-sayed?
```suggestion
ऐसे वर्कलोड्स के लिए जिन्हें लोकल स्टेट की आवश्यकता होती है, {{< glossary_tooltip text="स्टेटफुलसेट" term_id="StatefulSet" >}} का उपयोग करने पर विचार करें।
```

**Review by Babapool — COMMENTED** [review]
Some nits from my side, hope you find them useful. Thanks (Apart from these nits the PR looks LGTM)

**Comment by divya-mohan0209** [comment]
@hamees-sayed Thank you for your contribution. Due to lack of any response from your end, we will be closing this PR right now. However, if you feel that you're able to address the feedback provided and actively work on the PR, please feel free to reopen it & we shall begin right where we left off :)

/close

**Pull request closed without merging**